components/modal.jsx

```diff
@@ -5,7 +5,9 @@ var ModalManagerMixin = require('../mixins/modal-manager');
 var Modal = React.createClass({
   mixins: [React.addons.PureRenderMixin, ModalManagerMixin],
   propTypes: {
-    modalTitle: React.PropTypes.string
+    modalTitle: React.PropTypes.string,
+    foldedStyle: React.PropTypes.bool,
```

Comment:
Awesome!  I tried to do the same for another ticket but it didn't work(not sure why :eyes:). That's why I didn't bother using the approach you suggested this time.  Anyways, thanks for flagging this - all good now!  I'll update the code.